app/customer.py

```diff
@@ -0,0 +1,37 @@
+from math import dist
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list[int],
+                 money: float,
+                 car_consumption: float) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.home = location
+        self.location = location
+        self.money = money
+        self.shop = None
+        self.home = location
```

Comment:
self.shop = None - duplicated
customer shouldn't have a shop

Reply:
the line from the 29 to 39 sorts the cheapest shop for customers and self.shop was contain this cheapest shop for customers and compares self.shop with current shop while we sort them

Reply:
customer shouldn't have a shop in the concept of OOP, it's invalid
Fix it